### PR TITLE
Visitor can add products to an order

### PR DESCRIPTION
**Pull request by marizaswart**
### Description
* A visitor can add products to an order.
* The user can click on Add to Order button and the products will be added to their order with a confirmation notification displayed.
  
### What we have learned
* Not to over complicate tasks and go step by step with the features to solve one problem at a time.
* How to add multiple products to the same order.

### CODE REVIEW

File: app/controllers/charges_controller.rb [hunk 1]
```diff
@@ -19,7 +19,7 @@ def create
 
     if charge.paid?
       message = 'Your transaction was successful!'
-      redirect_to cart_path, notice: message
+      redirect_to charges_path, notice: message
```
Comment (tochman) [inline app/controllers/charges_controller.rb]: All of this seems out of scope. Why do you choose to keep this controller around?

File: app/views/charges/show.html.erb [hunk 2]
```diff
@@ -1,7 +1,7 @@
-<%= form_tag cart_path do %>
+<%= form_tag charges_path do %>
 <script src="https://checkout.stripe.com/checkout.js" class="stripe-button"
     data-key="<%=ENV['PUBLISHABLE_KEY'] %>"
     data-description="Payment for food"
     data-amount="1000"
     data-locale="auto"></script>
-<% end %>
\ No newline at end of file
+<% end %>
```
Comment (tochman) [inline app/views/charges/show.html.erb]: Out of scope IMO

File: app/views/partials/_navbar.html.erb [hunk 3]
```diff
@@ -1,17 +1,17 @@
...
+  </button>
+  <div class="navbar-collapse collapse" id="collapsing-navbar">
+    <ul class="navbar-nav">
+      <li class="nav-item active">
+        <%= link_to ' Home', root_path, class: 'glyphicon glyphicon-home' %>
+      </li>
+    </ul>
+    <ul class="navbar-nav ml-auto">
+      <li class="nav-item">
+        <%#= link_to ' Cart', orders_path, class: 'glyphicon glyphicon-shopping-cart', id: 'order-icon' %>
```
Comment (tochman) [inline app/views/partials/_navbar.html.erb]: Please get rid of this commented out line

**Review by tochman — CHANGES_REQUESTED** [review]
Not quite there yet....

File: config/routes.rb [hunk 4]
```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
-  get '/cart', to: 'cart#show'
-  post '/cart', to: 'cart#create'
+  get '/charges', to: 'charges#show'
+  post '/charges', to: 'charges#create'
```
Comment (tochman) [inline config/routes.rb]: Not sure why this is around. Out of scope.

**Review by tochman — CHANGES_REQUESTED** [review]

File: app/controllers/charges_controller.rb (hunk 1 above)
Comment (SophieSjoberg) [reply app/controllers/charges_controller.rb]: Yes it is, we are taking it out from this PR and making a separate PR for it 👍

File: app/views/charges/show.html.erb (hunk 2 above)
Comment (SophieSjoberg) [reply app/views/charges/show.html.erb]: Yes it is, we are taking it out from this PR and making a separate PR for it 👍

File: config/routes.rb (hunk 4 above)
Comment (SophieSjoberg) [reply config/routes.rb]: Yes it is, we are taking it out from this PR and making a separate PR for it 👍

**Review by tochman — APPROVED** [review]
I will okay this, but you still have code that mentions ‘cart’ (in step definitions). Good job team!
